packages/aws-cdk-lib/aws-appsync/README.md

```diff
@@ -423,7 +465,7 @@ const logConfig: appsync.LogConfig = {
 new appsync.GraphqlApi(this, 'api', {
   authorizationConfig: {},
   name: 'myApi',
-  schema: appsync.SchemaFile.fromAsset(path.join(__dirname, 'myApi.graphql')),
+  apiSource: appsync.ApiSource.fromFile(path.join(__dirname, 'myApi.graphql')),
```

Comment:
I know I came up with `apiSource` myself, but I'm not a 100% happy with it 😅 . Have you given any thought to alternatives? Is there a term that AppSync itself uses for that might apply here?

Reply:
Probably `schema` should be part of the name. Both GraphQL APIs and Merged APIs have a schema configuration in the AWS console:
- GraphQL API: _Design your schema using GraphQL SDL, attach resolvers, and quickly deploy AWS resources._
- Merged API: _Your Merged API’s schema is created from the schemas of your included Source APIs, and is read only. Changes to the schema must be initiated in your source APIs._

How about `schemaSource` or `schemaDefinition`?

@elthrasher @Lock128 After using the Merged API in our hackathon project: Do you have any feedback or ideas for a new name? We are looking for a better name than `apiSource` (see examples in the pull request).

Reply:
I like "definition". How about just `definition?: Definition` or `apiDefinition?: ApiDefinition` ?

Reply:
Let's call it `definition`. Other attributes like `name` don't use _api_ as a prefix, so I'm fine with definition.